sale_automatic_workflow/models/automatic_workflow_job.py

```diff
@@ -144,7 +165,10 @@ def _sale_done(self, sale_done_filter):
     def run_with_workflow(self, sale_workflow):
         workflow_domain = [("workflow_process_id", "=", sale_workflow.id)]
         if sale_workflow.validate_order:
-            self._validate_sale_orders(
+            _self = self.with_context(
+                send_order_confirmation_mail=sale_workflow.send_order_confirmation_mail
+            )
+            _self._validate_sale_orders(
                 safe_eval(sale_workflow.order_filter_id.domain) + workflow_domain
             )
```

Comment:
Well I meant somthing like
```suggestion
            self.with_context(
                send_order_confirmation_mail=sale_workflow.send_order_confirmation_mail
            )._validate_sale_orders(
                safe_eval(sale_workflow.order_filter_id.domain) + workflow_domain
            )
```